src/duplicacy_gcdstorage.go

```diff
@@ -45,20 +45,21 @@ type GCDConfig struct {
 	Token        oauth2.Token    `json:"token"`
 }
 
+func (storage *GCDStorage) computeInitialBackoff() float64 {
+        minimumInitialDelay := float64(storage.numberOfThreads) * 0.1
+        maximumInitialDelay := float64(storage.numberOfThreads) * 0.2
+        return rand.Float64() * (maximumInitialDelay - minimumInitialDelay + 1.0) + minimumInitialDelay
+}
+
 func (storage *GCDStorage) shouldRetry(threadIndex int, err error) (bool, error) {
-	const LIMIT_BACKOFF_TIME = 64
-	const MAX_NUMBER_OF_RETRIES = 15
-	minimumSleepRatio := 0.1
-	maximumSleepRatio := 0.2
-	minimumSleep := float64(storage.numberOfThreads) * minimumSleepRatio
-	maximumSleep := float64(storage.numberOfThreads) * maximumSleepRatio
-	rand.Seed(time.Now().UnixNano()) // unsure if this is needed
+	const MAX_BACKOFF_TIME = 64
```

Comment:
I have no reason for 32 or 64. I just like round numbers. Less than 32 though may be too small (for example in case of my router rebooting, i still want the backup to continue!)

Reply:
It doesn't matter whether it is 64 or 60.  The initial backoff isn't a whole number anyway.  64 is recommended here: https://cloud.google.com/storage/docs/exponential-backoff